README.asciidoc

```diff
@@ -59,5 +59,37 @@ The gRPC framework provides flexibility in the way services and their methods ar
 The data persisted by the connectors framework is distributed across the Fusion cluster. Each node holds its primary partition of the data, as well as backups of other partitions.
 If a node goes down during a crawl, the data store remains intact and usable. Connector implementations do not need to be concerned with this layer, because it is all handled by Fusion.
 
+== CrawlDB fields
+* Core fields required for any connector would include: id and state_s.
+
+* Connector specific values include the "fields" and "metadata" properties, which result in Solr document prefixed fields: field_ and meta_, respectively.
```

Comment:
Looks good. Are there any "field_" examples we can include that are common across all connectors like "body" or "title"?

Reply:
Hmm, there might be a `lastModified` field now? @wsolaligue, does that new field get indexed into crawl-db using `field_*`?

Reply:
@mwmitchell yes, all the entries on candidates' field map are indexed with the prefix `field_`, but if you mean the field used by the purge stray items, that one is `modifiedAt_l` without the `field_` prefix

Reply:
@puneetkhanal  In the `modifiedAt_l` documentation there is no mention of the  purge stray items feature that Willy was mentioning. Should we add a note about it?

Reply:
@wsolaligue Is the modifiedAt field part of content documents only? Or are they also on crawl-db docs? If only content docs, let's include the description of that in a separate PR (content doc fields vs crawl-db doc fields).

Reply:
@mwmitchell `modifiedAt_l` is only part of the crawldb-docs, this field is not propagated to the content docs